Finished preprocessing the CelebA dataset...
200600
torch.cuda.is_available(): True
Traceback (most recent call last):
  File "/home/yerinyoon/code/anonymousNet/starGAN/stargan.py", line 165, in <module>
    main(config)
  File "/home/yerinyoon/code/anonymousNet/starGAN/stargan.py", line 82, in main
    solver = Solver(celeba_loader, rafd_loader, config, wandb)
  File "/home/yerinyoon/code/anonymousNet/starGAN/model.py", line 163, in __init__
    self.device=torch.cuda.set_device(config.gpu_id)
  File "/home/yerinyoon/anaconda3/envs/stargan/lib/python3.9/site-packages/torch/cuda/__init__.py", line 326, in set_device
    torch._C._cuda_setDevice(device)
RuntimeError: CUDA error: invalid device ordinal